scripts/pic32cxsgpinctrl.py

```diff
@@ -0,0 +1,287 @@
+# Copyright (c) 2024 Microchip
```

Comment:
You should keep original copyright with the new one, even when you rename a file.